src/main/assembly/fakeroot.xml

```diff
@@ -103,6 +103,12 @@ Installed in a fakeroot directory structure.
             <outputDirectory>/etc/init.d</outputDirectory>
             <fileMode>0755</fileMode>
         </fileSet>
+	<!-- .info files -->
+        <fileSet>
+            <directory>src/main/infofiles</directory>
+            <outputDirectory>/etc/grid-security/certificates</outputDirectory>
```

Comment:
they are indeed Argus specific, in that they don't (and will not) come from IGTF, but they do have the status of policy .info files in the same way as the policy-igtf-classic.info. Having them in /etc/argus will not work therefore, since they would not be found. The  reason for having them in the first place, is to prevent us from needing separate permit lines for each individual policy (e.g. classic, mics, slcs and iota) but instead making a single entry with e.g. policy-aspen-birch-cedar-dogwood.
What we could do is put the file itself in /etc/argus/... and make a symlink into the /etc/grid-security/certificates directory.